### PR TITLE
Vr/avail adapter documentation

### PR DESCRIPTION
**Pull request by vibhurajeev**
# Description
Minor updates to tests and added documentation on submitting transactions to avail.

## Docs
The updated docs for demo app is in demo-rollup-avail/README.md

### CODE REVIEW

File: adapters/avail/src/service.rs [hunk 1]
```diff
@@ -157,9 +157,8 @@ impl DaService for DaProvider {
     }
 
     // Extract the blob transactions relevant to a particular rollup from a block.
-    // This method is usually (but not always) parameterized by some configuration option,
-    // such as the rollup's namespace on Celestia. If configuration is needed, it should be provided
-    // to the DaProvider struct through its constructor.
+    // NOTE: The avail light client is expected to be run in app specific mode, and hence the
```
Comment (aterentic) [inline adapters/avail/src/service.rs:160]: Maybe to make it clear that block can contain other transactions, e.g.
`// transactions in the block are already filtered by rollup.`

File: adapters/avail/src/service.rs [hunk 2]
```diff
@@ -169,10 +168,8 @@ impl DaService for DaProvider {
 
     // Extract the list blob transactions relevant to a particular rollup from a block, along with inclusion and
     // completeness proofs for that set of transactions. The output of this method will be passed to the verifier.
-    //
-    // Like extract_relevant_txs, This method is usually (but not always) parameterized by some configuration option,
-    // such as the rollup's namespace on Celestia. If configuration is needed, it should be provided
-    // to the DaProvider struct through its constructor.
+    // NOTE: The light client here is trusted to have completed DA sampling and verification of inlusion and soundness.
```
Comment (aterentic) [inline adapters/avail/src/service.rs:171]: Just `inlusion` -> `inclusion` :)

File: examples/demo-rollup-avail/README.md [hunk 3]
```diff
@@ -74,44 +73,41 @@ circumstances.
 
 ## Getting Started
 
-### Set up Celestia
+### Set up Avail node
 
-The current prototype runs against Celestia-node version `v0.7.1`. This is the version used on the `arabica` testnet
-as of Mar 18, 2023. To get started, you'll need to sync a Celestia light node running on the Arabica testnet
+The current prototype runs against Avail-node version `v1.6.0`. To get started, you'll need to run 
+a celestia node locally.
```
Comment (aterentic) [inline examples/demo-rollup-avail/README.md:79]: `a celestia` -> `avail`

File: examples/demo-rollup-avail/README.md [hunk 4]
```diff
@@ -74,44 +73,41 @@ circumstances.
...
-1. Build celestia's key management tool `make cel-key`
-1. Initialize the node: `celestia light init --p2p.network arabica`
-1. Start the node with rpc enabled. Our default config uses port 11111: `celestia light start --core.ip https://limani.celestia-devops.dev --p2p.network arabica --gateway --rpc.port 11111`. If you want to use a different port, you can adjust the rollup's configuration in rollup_config.toml.
-1. Obtain a JWT for RPC access: `celestia light auth admin --p2p.network arabica`
-1. Copy the JWT and and store it in the `celestia_rpc_auth_token` field of the rollup's config file (`rollup_config.toml`). Be careful to paste the entire JWT - it may wrap across several lines in your terminal.
-1. Wait a few minutes for your Celestia node to sync. It needs to have synced to the rollup's configured `start_height `671431` before the demo can run properly.
+1. Clone the repository: `git clone https://github.com/availproject/avail.git`.
+2. `cd avail`
+3. Checkout version v1.6.0: `git checkout v1.6.0`.
+4. Checkout the code at v0.7.1: `git checkout tags/v0.7.1`
```
Comment (aterentic) [inline examples/demo-rollup-avail/README.md:84]: What is `0.7.1`?

File: examples/demo-rollup-avail/README.md [hunk 5]
```diff
@@ -74,44 +73,41 @@ circumstances.
...
+5. Run the node with command: `cargo run --release -p data-avail -- --dev --tmp`
 
-Once your Celestia node is up and running, simply `cargo +nightly run` to test out the prototype.
+For more details check instructions at: https://github.com/availproject/avail/tree/v1.6.0#readme
+
+### Set up Avail light client
```
Comment (aterentic) [inline examples/demo-rollup-avail/README.md:89]: Numbering :)

**Review by aterentic-ethernal — APPROVED** [review]

**Pull request merged**